src/subdomains/core/buy-crypto/routes/buy/dto/buy-payment-info.dto.ts

```diff
@@ -70,6 +70,12 @@ export class BuyPaymentInfoDto extends BankInfoDto {
   @ApiProperty({ description: 'Maximum volume in target asset' })
   maxVolumeTarget: number;
 
+  @ApiProperty({ description: 'Exchange rate in source/target' })
+  exchangeRate: number;
+
+  @ApiProperty({ description: 'Rate in sourceAmount/targetAmount' })
+  rate: number;
```

Comment:
`@ApiProperty({ description: 'Final rate (incl. fees) in source/target' })`
(same for sell + crypto)